util/types/Cargo.toml

```diff
@@ -10,7 +10,7 @@ molecule = "=0.6.0"
 ckb-fixed-hash = { path = "../fixed-hash" }
 numext-fixed-uint = { version = "0.1", features = ["support_rand", "support_heapsize", "support_serde"] }
 bytes = { version="0.5.4", features = ["serde"] }
-merkle-cbt = "0.2"
+merkle-cbt = { git = "https://github.com/nervosnetwork/merkle-tree", branch = "quake/tree-verify-proof" }
```

Comment:
waiting for https://github.com/nervosnetwork/merkle-tree/pull/20, will use a published crate version later